src/store/actions/index.js

```diff
@@ -1,2 +1,3 @@
 export const LOGIN = "LOGIN";
 export const FETCH_USERS = "FETCH_USERS";
+export const HACK_A_THON = "HACK_A_THON";
```

Comment:
This action name doesnt tell me what the action is doing (eg, `FETCH_HACKATHON_DETAILS`)